src/main/java/cr0s/warpdrive/block/TileEntityAirGenerator.java

```diff
@@ -2,12 +2,20 @@
 
 import net.minecraft.block.Block;
 import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.util.MathHelper;
 import net.minecraftforge.common.util.ForgeDirection;
+import cpw.mods.fml.common.Optional;
 import cr0s.warpdrive.WarpDrive;
 import cr0s.warpdrive.config.WarpDriveConfig;
+import dan200.computercraft.api.lua.ILuaContext;
+import dan200.computercraft.api.peripheral.IComputerAccess;
+import li.cil.oc.api.machine.Arguments;
+import li.cil.oc.api.machine.Callback;
+import li.cil.oc.api.machine.Context;
 
 public class TileEntityAirGenerator extends TileEntityAbstractEnergy {
 	private int cooldownTicks = 0;
+	private boolean generatorEnabled = true;
```

Comment:
This is a air generator, no need to remind it in the property name here. Change it to "isEnabled" to be consistent with the rest of the mod.